tests/TestFormula.elm

```diff
@@ -222,7 +224,8 @@ testParseSigned =
 
 parseTests =
     describe "parse tests"
-        [ test "Neg " <| \() -> testParse "- a" <| Neg a
+        [ test "Eq" <| \() -> testParse "(d≐j)" <| EqAtom d j 
+        , test "Neg " <| \() -> testParse "- a" <| Neg a
```

Comment:
please add: equality without parentheses; formula with unparenthesized negated equality (e.g., `- x = c`) connected by a connective to something; quantified formulas with atomic (e.g. `∃x f(x) = x`) and complex subformulas (`\forall x \exists z(∀y f(x) = g(x,y) -> ¬∀y -p(x, y, z))`).